src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -69,6 +70,34 @@ CaseDetails submitForCaseworker(
             @RequestParam Map<String, Object> searchCriteria
     );
 
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/caseworkers/{userId}/jurisdictions/{jurisdictionId}/case-types/{caseType}/cases" +
+                    "/pagination_metadata",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE)
```

Comment:
Is this needed? i.e. a GET request shouldn't require a content type, there are some APIs in CCD which already do this incorrectly which is why some of the requests here have it

Reply:
Its not intentional its because its set at the class level on the controller
https://git.reform.hmcts.net/case-management/case-data-store-app/blob/develop/src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java#L42

If setting on the class level it needs to be reset on the method level to the correct one like:
https://git.reform.hmcts.net/case-management/case-data-store-app/blob/develop/src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseAccessEndpoint.java#L29